### ISSUE TITLE
Scoring result divide by rounds distances.

### ISSUE THREAD

**Issue by vijai3d** [archscorer/archscorer#36]
Problem.

Currently, in Event Results table we can see the only total sum of scoring for all distances of round. 

Proposal.
Adds coring column for each round distance. Sum column leave as it is.

**Comment by adlerpriit** [comment]
Not sure what is it that you propose here? Different rounds scores are shown separately. Some rounds are traditionally scored half and half (in arcschorer course type "unit"), and these are shown as sum to represent the "round" score.

**Comment by vijai3d** [comment]
Yes, this ticket should be investigated in more details. Lets leave as is
right now. I will discribe in more details later.

вт, 5 апр. 2022 г., 21:30 Priit Adler ***@***.***>:
